submissions/OlgaBarnikova/Popup/index.html

```diff
@@ -0,0 +1,119 @@
+                                    <div>Maps</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/documents.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Documents</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/blogs.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Blogs</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/apps.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Apps</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/news.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>News</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/search.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Search</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/promo.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Promo</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/tanslation.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Translation</div>   
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_block_img"><img src="popup-icons/videos.png" alt="" 
+                                        class="popup_img" /></a>  
+                                    <div>Videos</div>   
+                                </li>
+                            </ul>
+                            <label for="popup-more">
+                                <div class="popup_more_name">more...</div>                      
+                            </label>
+                            <input type="checkbox" id="popup-more" class="popup_input_more" />
```

Comment:
Add styles on focus state, also, hide this button when the second part of the popup is open. [Watch video](https://share.vidyard.com/watch/ZDCPyrd5EAnKvP895smxQc?)